com.gitlab.inendi.Inspector.yaml

```diff
@@ -20,4 +20,4 @@ modules:
     sources:
       - type: archive
         url: https://inendi.gitlab.io/inspector/inendi-inspector_flatpak.tar.bz2
-        sha256: 4218e69b40eba2fcc90897482ea811dc1f088b2c33084d73d87b232796fd0146
+        sha256: b1609d98ca0a9f7e70a259be8e6f84dae31eeb6bdb8f720b4dfb536669ea5a99
```

Comment:
The tarball URL should be (have been) versioned. Otherwise you are making the build not reproducible.

Also you are building with an EOL runtime. I remember during the review mentioning you were alreasdy using an old runtime. I guess it didn't change.

Build failed because changing the publishing delay is not allowed by default.

Reply:
I can try to version the tarball URL.

I updated the runtime to 22.08, I just forgot to update the ref in the manifest file...

Why changing the publishing delay is not allowed "by default" ? What should be done to be able to do that ? The flathub.json file commited in master has it set to "0"  though.

Reply:
> Why changing the publishing delay is not allowed "by default" ?

This was missed in the initial review. We now have a linter that will catch it and Error.